test/integration/narrow/routes/endemics/date-of-visit-ms.test.js

```diff
@@ -255,17 +255,60 @@ describe('POST /claim/endemics/date-of-visit handler', () => {
 
     const $ = cheerio.load(res.payload)
     expect(res.statusCode).toBe(400)
-    expect($('p.govuk-error-message').text().trim()).toEqual(
-      'Error: The date of review must be a real date'
-    )
+    expect($('#main-content > div > div > div > div > div > ul > li > a').text().trim()).toEqual('Enter a date in the boxes below')
```

Comment:
this selector feels a bit fragile, might be worth sticking a `data-testid` attribute on it if we want to test it?

Reply:
Yeah fair point - I'm half tempted to start using snapshots to ensure the html is the same

Reply:
Snapshots removed - using a better xpath now